automation_tools/__init__.py

```diff
@@ -1126,6 +1129,8 @@ def product_install(distribution, create_vm=False, certificate_url=None,
     :param str certificate_url: where to fetch a fake certificate.
 
     """
+    # Fetch the Satellite Version information.
+    satellite_version = os.environ.get('SATELLITE_VERSION')
```

Comment:
```satellite_version``` assignment was already there I just shifted it to the beginning of product_install()
for downstream we are not failing anyway, better then fail is to assume 6.2 if not defined